src/ML-METATOMIC/pair_metatomic.h

```diff
@@ -34,6 +34,7 @@ PairStyle(metatomic_9, PairMetatomic);
 #include "pair.h"
 
 #include <vector>
+#include <torch/torch.h>
```

Comment:
We should not include anything in the `pair_xxx` header (this is why we have a separate `PairMetatomicData` containing anything that is not pure LAMMPS.